lib/ansible/config/base.yml

```diff
@@ -897,10 +897,14 @@ DEFAULT_NO_LOG:
 DEFAULT_NO_TARGET_SYSLOG:
   name: No syslog on target
   default: False
-  description: Toggle Ansible logging to syslog on the target when it executes tasks.
+  description:
+  - Toggle Ansible logging to syslog on the target when it executes tasks. On Windows hosts this will disable a newer
+    style PowerShell modules from writting to the event log.
   env: [{name: ANSIBLE_NO_TARGET_SYSLOG}]
   ini:
   - {key: no_target_syslog, section: defaults}
+  vars:
+  - {name: ansible_no_target_syslog}
```

Comment:
version_added

Reply:
I can't add the `version_added` key because the option has always been there. I can add an entry to the description saying this was added. Or can you do `version_added` for a vars entry?

Reply:
version added can be added per env var, variable or even ini entry, also you can deprecate the whole config or just one of the ways to set it

```
 vars:
  - name: ansible_no_target_syslog
    version_added: '2.10'
 ￼```

Reply:
Ah cool, didn't know we could do it per entry and thought it was per option, will update it.